var/spack/repos/builtin/packages/py-scikit-learn/package.py

```diff
@@ -81,6 +81,26 @@ def setup_build_environment(self, env):
             env.append_flags(
                 'LDFLAGS', self.spec['llvm-openmp'].libs.ld_flags)
 
+    def install_args(self, spec, prefix):
+        return (super(PyScikitLearn,self).install_args(spec, prefix)
+                + ['--skip-build'])
+
+    def build(self, spec, prefix):
+        """Install everything from build directory."""
+        args = (spec['py-numpy'].package.fortran_args(spec, prefix)
```

Comment:
There are a LOT of packages that depend on numpy. Is this hack only needed for scikit-learn/scipy or is it something that most packages need?